packages/launch/src/focused-launch/style.scss

```diff
@@ -1,3 +1,12 @@
+
+@import '~@wordpress/base-styles/mixins';
+@import '~@wordpress/base-styles/variables';
+@import '~@wordpress/base-styles/breakpoints';
+
 .focused-launch-container {
-    padding: 0 10px;
+	padding: 0 20px;
+
+	@include break-medium {
+		padding: 0 10px;
```

Comment:
Why is the padding being reduced on larger screens?

Reply:
Thats what happens on the [summary view](https://github.com/Automattic/wp-calypso/blob/master/packages/launch/src/focused-launch/summary/style.scss#L47-L54) so I am keeping that spacing consistent